### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_AllJobs_ScreenReader:NVDA is not announcing the information represented by tick image.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5750]
**"Bug Description**:
[Check out Keros!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
 GitHubTags: #A11yMAS; #A11yTCS; #MAS1.3.2; #A11y_AzureDataStudio; #Win32; #DesktopApp
**Environment Details:**
Application name: Azure Data Studio (Agent, Create External Data, Notebooks)
App Version: 1.8.0-insider (user setup)
VS Code 1.33.1
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362
**Additional Details:**
Tool Name: NVDA
Version :2019.1
**MAS Reference:MAS1.3.2**
**Repro Steps:**
1.Install the app Azure Data Studio.
2.Naviagte to the SQL agent tab->Jobs->Click on any the jobs in the table.
3.Navigate with tab till steps table.
4.Start NVDA 
5.observed the announcement of screen reader (NVDA)
**Actual:**
NVDA is  not announcing the information represented by tick image. NVDA announcement starts from step id column.

![image](https://user-images.githubusercontent.com/39204909/58859640-de6fe600-86c7-11e9-971d-3b0c6d7d976f.png)

**Expected:**
NVDA should announce each and every information which is inside tick image.
**User Impact:**
User with vision and motor disabilities will miss the information regarding that image and will deviate from the actual data.
**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference:1.3.1**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
**Attachment for Reference:**
[5750_AccessibilityA11y_AzureDataStudio_AllJobs_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3237083/5750_AccessibilityA11y_AzureDataStudio_AllJobs_ScreenReader.pptx)
Does this issue occur when all extensions are disabled?: Yes
